doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -238,16 +238,16 @@ Rewriting with Leibniz and setoid equality
    :name: rewrite *; _
    :undocumented:
 
-.. tacn:: replace @one_term__from with @one_term__to {? @occurrences } {? by @ltac_expr3 }
+.. tacn:: replace {? {| -> | <- } } @one_term__from with @one_term__to {? @occurrences } {? by @ltac_expr3 }
           replace {? {| -> | <- } } @one_term__from {? @occurrences }
    :name: replace; _
 
    The first form replaces all free occurrences of :n:`@one_term__from`
    in the current goal with :n:`@one_term__to` and generates an equality
```

Comment:
This sounds a bit confusing and ambiguous to me, as this may imply that this first "form" is either `replace OPT <- @one_term__from with @one_term__to {? @occurrences }` or `replace OPT <- @one_term__from {? @occurrences }`, as opposed to the variants with a `->`.

Reply:
I changed it to a "when" subordinating conjunction instead.